src/GifPickerReact.css

```diff
@@ -1,12 +1,27 @@
-.GifPickerReact {
+body[data-theme="light"] {
     --gpr-highlight-color: #007aeb;
     --gpr-hover-bg-color: #f1f8ff;
     --gpr-focus-bg-color: #e0f0ff;
     --gpr-text-color: #858585;
     --gpr-search-input-bg-color: #f6f6f6;
     --gpr-picker-border-color: #e7e7e7;
+    --gpr-cat-font-color: #e7e7e7;
     --gpr-bg-color: #fff;
+}
+
+body[data-theme="dark"] {
+    --gpr-highlight-color: #8961a7;
+    --gpr-hover-bg-color: #858688;
+    --gpr-focus-bg-color: #9a9a9b;
+    --gpr-text-color: #f6f6f6;
+    --gpr-search-input-bg-color: #3b3b3b;
+    --gpr-picker-border-color: #2b2b2b;
+    --gpr-bg-color: rgb(42, 41, 41);
+    --gpr-cat-font-color: #f6f6f6;
+}
+
```

Comment:
Yeah you rely on body theme and theme configuration prop is ignored, Look how it's done here https://github.com/ealush/emoji-picker-react/blob/master/src/EmojiPickerReact.css

Reply:
Ah ok, will update to this instead

Reply:
This is updated to use similar to EmojiPicker but without clsx or classnames dep package